### PR TITLE
Fix Auth0Service::getUserProfileByA0UID() phpdoc

### PR DESCRIPTION
**Pull request by olix21**
### Changes

The documentation gave a wrong return type-hint.
It's now fixed! 😄

### CODE REVIEW

File: src/Security/Auth0Service.php [hunk 1]
```diff
@@ -55,7 +55,9 @@ public function __construct($api_secret, $domain, $api_identifier, $authorized_i
     /**
      * Get the Auth0 User Profile based on the JWT (and validate it).
      *
-     * @return User info as described in https://docs.auth0.com/user-profile
+     * @param string $jwt
```
Comment (joshcanhelp) [inline src/Security/Auth0Service.php]: Missing a description here

File: src/Security/Auth0Service.php [hunk 2]
```diff
@@ -55,7 +55,9 @@ public function __construct($api_secret, $domain, $api_identifier, $authorized_i
...
+     *
+     * @return array info as described in https://docs.auth0.com/user-profile
```
Comment (joshcanhelp) [inline src/Security/Auth0Service.php]:
```suggestion
     * @return array info as described in https://auth0.com/docs/users/concepts/overview-user-profile
```

File: src/Security/Auth0Service.php (hunk 1 above)
Comment (joshcanhelp) [inline src/Security/Auth0Service.php]: Missing a parameter description here

Comment (olix21) [reply src/Security/Auth0Service.php]: I've added a description

**Pull request merged**